rust/c509-certificate/examples/cli/main.rs

```diff
@@ -233,9 +233,9 @@ fn validate_certificate_type(
     self_signed: bool, certificate_type: Option<u8>,
 ) -> anyhow::Result<()> {
     if self_signed && certificate_type.unwrap_or(SELF_SIGNED_INT) != SELF_SIGNED_INT {
-        return Err(anyhow::anyhow!(
-            "Certificate type must be 0 if self-signed is true"
-        ));
+        return Err(anyhow::anyhow!(format!(
```

Comment:
I am not mistaken `anyhow!` macro has string formatting under the hood, so using `format!` most probable is not necessary.